src/opnsense/mvc/app/controllers/OPNsense/Dnsmasq/forms/dialogDomainOverride.xml

```diff
@@ -33,6 +33,12 @@
         <type>text</type>
         <help>Source IP address for queries to the DNS server for the override domain. Best to leave empty</help>
     </field>
+    <field>
+        <id>domainoverride.ipset</id>
+        <label>Ipset</label>
+        <type>dropdown</type>
+        <help>Choose an "external (advanced)" type alias from "Firewall - Aliases". Whenever a client successfully resolves the domain in this override, the resolved IP addresses will be automatically added to the chosen alias. You can create multiple domain overrides to add multiple domains to the same alias. Adding a wildcard domain will also add all IP addresses of resolved subdomains. Please note that domain TTL is not evaluated; once an IP address is added, it will stay permanently, or until manually flushed in "Firewall - Diagnostics - Aliases".</help>
```

Comment:
```suggestion
        <help>Choose an "external (advanced)" type alias from "Firewall - Aliases". Whenever a client successfully resolves the domain in this override, the resolved IP addresses will be automatically added to the chosen alias. You can create multiple domain overrides to add multiple domains to the same alias. Adding a wildcard domain will also add all IP addresses of resolved subdomains. Please note that DNS record TTL is not evaluated; once an IP address is added, it will stay permanently, or until manually flushed in "Firewall - Diagnostics - Aliases".</help>
```